### PR TITLE
typo: derivation specs

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Use `setL1BlockIsthmus` rather than than `setL1BlockInterop`

### CODE REVIEW

File: specs/interop/derivation.md [hunk 1]
```diff
@@ -35,7 +34,7 @@ if any exists, or after the first L1 attributes transaction if there are no user
 
 The order of deposit transactions occurs as follows:
 
-1. L1 attributes transaction calling [`setL1BlockValuesInterop()`](../protocol/ecotone/l1-attributes.md).
+1. L1 attributes transaction calling [`setL1BlockValuesIsthumus()`](../protocol/ecotone/l1-attributes.md).
```
Comment (ajsutton) [inline specs/interop/derivation.md]: Extra u in Isthmus:
```suggestion
1. L1 attributes transaction calling [`setL1BlockValuesIsthmus()`](../protocol/ecotone/l1-attributes.md).
```

Comment (tynes) [reply specs/interop/derivation.md]: Good catch, thank you!

**Review by ajsutton — APPROVED** [review]

**Comment by Java7981** [comment]
Gooooood 🎉 🎉 🎉

**Comment by Java7981** [comment]
Nice

**Comment by Java7981** [comment]
Nat

**Pull request merged**